src/lib/parse/blueprint.js

```diff
@@ -28,7 +41,36 @@ module.exports = function(filePath, autoOptions, routeMap) {
 
             cb();
 
-            function setupResourceAndUrl(resource) {
+            function validateAndSetupResource(fixtureResource: BlueprintResource, contract: Contract) {
+                const fixtureUrl = urlParser.parse(fixtureResource.uriTemplate).url;
+               
+                const contractActions: ?Actions = getActions(fixtureUrl, contract.resources);
+
+                let valdatedResource = contracts.removeInvalidActions(fixtureResource, contractActions);
```

Comment:
rename to `removeInvalidFixture`. add comment that single action may contain multiple fixtures.